packages/core/src/mm/objects/gameplay_keep.S

```diff
@@ -0,0 +1,33 @@
+#include <combo.h>
+
+/* Hammer Animations - Patch AnimHeader Frame Count */
+
+/* gPlayerAnim_Link_otituku_w -> gPlayerAnim_link_hammer_hit */
```

Comment:
I'm just curious, which unused animations did these replace?

Reply:
iirc it was animations for talking to Navi/Zelda after tower escape in OoT, then turning and covering Zelda when the pile of rubble makes a noise. Also looking up at the lowering crystal containing Zelda after beating Ganondorf.